microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -99,43 +99,41 @@ When you ran the optimized version of the page, the blue icons move faster.  Why
 
 Once you have a recording of the page's performance, you can assess the page's performance and find the cause of any performance issues.
 
-1. The **CPU** chart is displayed along the top.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The **CPU** chart shows that these regions make up a large area, meaning that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that you should find ways to do less work.
+1. The **CPU** chart is displayed along the top.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The **CPU** chart shows that these regions make up a large area, meaning that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that the page is not performing well.
 
    ![The CPU chart and Summary panel.](../media/evaluate-performance-performance-cpu-chart.msft.png)
 
 1. Hover over the **CPU** or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of the performance recording.
 
    ![Hover on a frame.](../media/evaluate-performance-performance-frame-hover.msft.png)
 
-The display indicates that the webpage isn't performing well.  In real scenarios, it may not be so clear whether the page is performing well, so having all of the tools to make measurements comes in handy.
 
 
-#### Bonus: Open the FPS meter
+#### Bonus: Open the Frame Rendering Stats overlay
 
-Another handy tool is the FPS meter, which provides real-time estimates for FPS as the page runs.
+Another handy tool is the **Frame Rendering Stats** overlay, which provides real-time estimates for FPS as the page runs. The **Frame Rendering Stats** overlay is not required for this tutorial but may provide helpful insight.
 
-1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the **Command Menu**.
+1. In DevTools, press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the **Command Menu**.
 
 1. Start typing `Rendering` in the **Command Menu** and click **Show Rendering**.
 
-1. In the **Rendering** tool, turn on **FPS Meter**.  A new overlay appears in the top-right of your viewport.
+1. In the **Rendering** tool, turn on **Frame Rendering Stats**.  A new overlay appears in the top-left of your webpage.
 
-   ![The FPS meter.](../media/evaluate-performance-fps-meter-overlay.msft.png)
+   ![The FPS overlay.](../media/evaluate-performance-fps-meter-overlay.msft.png)
 
-1. Turn off the **FPS Meter** and press `Escape` to close the **Rendering** tool.  You aren't using **FPS Meter** in this tutorial.
+1. Turn off **Frame Rendering Stats** by deselecting the checkbox.
```

Comment:
```suggestion
1. Clear the **Frame Rendering Stats** checkbox.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/check-checkbox-check-mark

Reply:
```suggestion
1. When you are done reviewing the FPS data, clear the **Frame Rendering Stats** checkbox to hide the overlay.
```
This helps indicate that hiding the overlay is an optional step based on the user's preference and not required to complete the tutorial.

Reply:
ok.  I don't have a Resolve Conversation button, please click that button as appropriate.  I don't have a Commit button.